modules/svdb/merge/main.nf

```diff
@@ -22,8 +22,14 @@ process SVDB_MERGE {
     def args   = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
     def input  = ""
-    for (int index = 0; index < vcfs.size(); index++) {
-        input += " ${vcfs[index]}:${priority[index]}"
+    if(priority) {
+        prio = "--priority ${priority.join(',')}"
```

Comment:
Nice, I think with this you can remove `--priority ${priority.join(',')} \\ `in the script block right?

Reply:
Yupe! Good catch!